lib/js/components/SurveyQuestions/SurveyQuestionOpenEnded/SurveyQuestionOpenEnded.vue

```diff
@@ -79,13 +80,13 @@
 </style>
 
 <script lang="ts">
-import DsCard from '../../Cards/Card';
+import DsCard from '../../Cards/Card/Card';
```

Comment:
we should be consisten and use `index.ts`. It means, we should revert this line

Reply:
True, fixed.